docs/reference/utility-apis/IdentityApi.md

```diff
@@ -1,7 +1,7 @@
 # IdentityApi
 
 The IdentityApi type is defined at
-[packages/core-api/src/apis/definitions/IdentityApi.ts:22](https://github.com/backstage/backstage/blob/ca535f2f66c3a4980c80f4b1a049dfd07569010e/packages/core-api/src/apis/definitions/IdentityApi.ts#L22).
+[packages/core-api/src/apis/definitions/IdentityApi.ts:22](https://github.com/backstage/backstage/blob/master/packages/core-api/src/apis/definitions/IdentityApi.ts#L22).
```

Comment:
Thanks!

Since the link points to a specific line number, I think it makes sense to keep the permalink behavior with

`https://github.com/backstage/backstage/blob/abd85269fbea466c18a71fa661335c0713a32c07/packages/core-api/src/apis/definitions/IdentityApi.ts#L22`
https://github.com/backstage/backstage/blob/abd85269fbea466c18a71fa661335c0713a32c07/packages/core-api/src/apis/definitions/IdentityApi.ts#L22

Reply:
Could you clarify what you mean? I did keep the permalink behavior "#L22"? or.. ? 👀  you mean you wanted me to still point to a specific commit so that the content always will be there when accessing the link?

Reply:
The later one, with this link :)

`https://github.com/backstage/backstage/blob/abd85269fbea466c18a71fa661335c0713a32c07/packages/core-api/src/apis/definitions/IdentityApi.ts#L22`